ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -216,6 +260,54 @@ export default class StatusDropdown extends React.Component {
     alertStatus === 'investigating' ||
     (alertStatus !== status && this.isResolved(alertStatus));
 
+  calculateDueDate(created) {
+    const createdAt = new Date(created);
+    const dueDate = new Date(created);
+    dueDate.setDate(dueDate.getDate() + 3);
+    const numberOfNonWorkingDays = this.getNumberOfNonWorkingDays(
+      createdAt,
+      dueDate,
+    );
+
+    dueDate.setDate(
+      numberOfNonWorkingDays !== 0
+        ? dueDate.getDate() + numberOfNonWorkingDays
+        : dueDate.getDate(),
```

Comment:
I don't think it's necessary to set the `dueDate` again here if `numberOfNonWorkingDays` is 0 (the `dueDate` was already set on line 266).